entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/impl/EntityRecordImpl.java

```diff
@@ -81,4 +83,14 @@ public void setDbId(long dbId_param) {
 	public long getDbId() {
 		return dbId;
 	}
+
+	@Override
+	public boolean getDisabled() {
```

Comment:
the disabled field is not part of the serialization, jsonIgnore annotations needs to be added here

Reply:
Done.